packages/subgraph/tests/admin/adminMembers.test.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  assert,
+  clearStore,
+  dataSourceMock,
+  test,
+  describe,
+  beforeEach,
+  afterEach,
+  logStore
+} from 'matchstick-as/assembly/index';
+
+import {ADDRESS_ONE, ADDRESS_TWO, DAO_ADDRESS} from '../constants';
+import {createGrantedEvent, createRevokedEvent} from './utils';
+import {
+  handleGranted,
+  handleRevoked
+} from '../../src/packages/admin/adminMembers';
+import {DataSourceContext} from '@graphprotocol/graph-ts';
+import {Administrator, AdministratorAdminPlugin} from '../../generated/schema';
+
+describe('AdminMembers', function() {
+  const AdminPermission =
+    '0xf281525e53675515a6ba7cc7bea8a81e649b3608423ee2d73be1752cea887889';
```

Comment:
can't we use `crypto.keccak256` ? or if not, the comment would be good that it's a hash of EXECUTE_PERMISSION

Reply:
yep makes sense. I will add it